### PR TITLE
fix(colors): ToggleSwitch colors fixed/updated

### PR DESCRIPTION
**Pull request by NicolasChampagne**
GitHub Issue: #11 #122 

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Refactor

## Description
 - Implements proper brushes
- Removes todos

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Tested UWP
- [ ] Tested iOS
- [x] Tested Android
- [ ] Tested WASM
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Material/Styles/Application/Colors.xaml [hunk 1]
```diff
@@ -28,6 +28,10 @@
 			<Color x:Key="OnErrorColor">#000000</Color>
 			<Color x:Key="OverlayColor">#51000000</Color>
 
+			<!-- Needed for android thumbtints -->
+			<Color x:Key="PrimaryDisabledVariantLightColor">#E9E5FA</Color>
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Application/Colors.xaml:32]: How will the dev override those? Can they be based off exising colors from the palette?

File: src/samples/Uno.Material.Samples/Uno.Material.Samples.Droid/Resources/values/Styles.xml [hunk 2]
```diff
@@ -10,5 +10,9 @@
 		
 		<!-- SplashScreen -->
 		<item name="android:windowBackground">@drawable/Splash</item>
+
+		<!-- Color style for toggle switch -->
```
Comment (MatFillion) [inline src/samples/Uno.Material.Samples/Uno.Material.Samples.Droid/Resources/values/Styles.xml:14]: that's in the sample, is it required by the consuming app?

File: src/library/Uno.Material/Styles/Application/Colors.xaml (hunk 1 above)
Comment (NicolasChampagne) [reply src/library/Uno.Material/Styles/Application/Colors.xaml:32]: These colors can't have an opacity value, therefore the can't be derived from the palette. Maybe design can add them as special colors

Comment (NicolasChampagne) [reply src/library/Uno.Material/Styles/Application/Colors.xaml:32]: These colors can't have an opacity value, therefore the can't be derived from the palette. Maybe design can add them as special colors

File: src/samples/Uno.Material.Samples/Uno.Material.Samples.Droid/Resources/values/Styles.xml (hunk 2 above)
Comment (NicolasChampagne) [reply src/samples/Uno.Material.Samples/Uno.Material.Samples.Droid/Resources/values/Styles.xml:14]: added to readme

**Pull request merged**